content/doc/book/installing/_master-agent-machines-setup-windows.adoc

```diff
@@ -0,0 +1,61 @@
+[[master-agent-setup-windows]]
+== Windows Agents
+
+Note: There is no need to install Jenkins on the agent machine.
+
+. On your master machine go to *Manage Jenkins* > *Manage Nodes*. +
+image:/images/installing/ManageNodes.png[Manage nodes,width=80%]
+
+. *New Node*
+.. *Enter Node Name*
+.. *Select Permanent Agent*
+.. Press *OK*. +
+image:/images/installing/NewNode.png[New node,width=40%]
+
+. Fill out the following:
+.. Set a *number of executors*
+... (one or more) as needed.
+.. Set a *Remote FS Root*
+... a home directory for the master on the agent machine.
+... For a _Windows agent_, use something like: "C:\Jenkins\"
+.. Select the appropriate *Usage* setting:
+... For an additional worker: _Utilize this node as much as possible_
+... For specialized jobs: _Leave this machine for tied jobs only_
+.. *Launch Method*:
+... An easy way to control a Windows agent is by using _Launch agent via Java Web Start_  (Recommended for Windows)
+... TODO: add steps for other methods.
```

Comment:
Need the content for TODO or should place it in a comment.  We don't want to publish pages with a TODO if we can avoid them.
```suggestion
... TODO: add steps for other methods.
```